elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HQLTransaction.java

```diff
@@ -153,7 +153,7 @@ else if (filters != null) {
                     query = session.createFilter(collection, "");
                 }
                 query.setFirstResult(pagination.getOffset());
-                query.setMaxResults(pagination.getLimit());
+                query.setMaxResults(pagination.getSize());
```

Comment:
Ideally, we shouldn't have to change the data stores.  I have some ideas for how this might work without requiring changes.